Dockerfile

```diff
@@ -9,5 +9,7 @@ RUN apk add --no-cache git
 COPY --from=src-cli /usr/bin/src /usr/bin
 RUN npm install -g @sourcegraph/lsif-tsc@${TAG}
 
-ENTRYPOINT []
```

Comment:
Is there a reason this was removed explicitly?

Reply:
Default entrypoint executes the following, which gives the same behaviour
```shell
#!/bin/sh
set -e

if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ]; then
  set -- node "$@"
fi

exec "$@"
```